samwise.js

```diff
@@ -1,54 +1,51 @@
 /**
```

Comment:
SCRIPT-Good job on updating the documentation along with the changes. It's very thorough and helpful!